Sem3_Homework/task1.py

```diff
@@ -0,0 +1,6 @@
+# Задайте список из нескольких чисел. Напишите программу, которая найдёт сумму элементов списка, стоящих на нечётной позиции.
+
+lst = [45, 2, 77, 3, 34, 1, 84]
+
+print("Сумма = ",
+      sum(lst[i] for i in range(1, len(lst)-1, 2)))

```

Comment:
```suggestion
      sum(lst[i] for i in range(1, len(lst), 2)))
```
Почему вы скучаете последний индекс? Ведь, по сути, длина списка может быть любой.